recipes/chef-fun/all/conanfile.py

```diff
@@ -0,0 +1,117 @@
+#        # Direct dependencies of header only libs are always transitive since they are included in public headers
+#        self.requires("dependency/0.8.1", transitive_headers=True)
+
+    def source(self):
+        # Download source package and extract to source folder
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+
+    # Not mandatory when there is no patch, but will suppress warning message about missing build() method
+    def build(self):
+        # The attribute no_copy_source should not be used when applying patches in build
+        apply_conandata_patches(self)
+
+    # Copy all files to the package folder
+    def package(self):
+        copy(self, pattern="LICENSE", dst=os.path.join(self.package_folder, "licenses"), src=self.source_folder)
+        copy(
+            self,
+            pattern="*.hh",
+            dst=os.path.join(self.package_folder, "include"),
+            src=os.path.join(self.source_folder, "include"),
+        )
+        copy(
+            self,
+            pattern="*.tcc",
+            dst=os.path.join(self.package_folder, "include"),
+            src=os.path.join(self.source_folder, "include"),
+        )
+
+    def package_info(self):
+        # Folders not used for header-only
+        self.cpp_info.bindirs = []
+        self.cpp_info.libdirs = []
+
+        # Set these to the appropriate values if the package has an official FindPACKAGE.cmake
+        # listed in https://cmake.org/cmake/help/latest/manual/cmake-modules.7.html#find-modules
+        # examples: bzip2, freetype, gdal, icu, libcurl, libjpeg, libpng, libtiff, openssl, sqlite3, zlib...
+        self.cpp_info.set_property("cmake_module_file_name", "ChefFun")
+        self.cpp_info.set_property("cmake_module_target_name", "ChefFun::ChefFun")
+        # Set these to the appropriate values if package provides a CMake config file
+        # (package-config.cmake or packageConfig.cmake, with package::package target, usually installed in <prefix>/lib/cmake/<package>/)
+        self.cpp_info.set_property("cmake_file_name", "chef-fun")
+        self.cpp_info.set_property("cmake_target_name", "chef-fun::chef-fun")
+        # Set this to the appropriate value if the package provides a pkgconfig file
+        # (package.pc, usually installed in <prefix>/lib/pkgconfig/)
+        self.cpp_info.set_property("pkg_config_name", "chef-fun")
+
+        ## Add m, pthread and dl if needed in Linux/FreeBSD
+        #if self.settings.os in ["Linux", "FreeBSD"]:
+        #    self.cpp_info.system_libs.extend(["dl", "m", "pthread"])
+
+        # TODO: to remove in conan v2 once cmake_find_package_* generators removed
+        self.cpp_info.names["cmake_find_package"] = "chef-fun"
+        self.cpp_info.names["cmake_find_package_multi"] = "chef-fun"
+        self.cpp_info.filenames["cmake_find_package"] = "chef-fun"
+        self.cpp_info.filenames["cmake_find_package_multi"] = "chef-fun"
```

Comment:
https://gitlab.com/libchef/chef-fun/-/blob/main/CMakeLists.txt

```suggestion
        self.cpp_info.libdirs = []
```